benchmark/Microsoft.IdentityModel.Benchmarks/JsonWebKeyComparison.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if NET6_0_OR_GREATER
```

Comment:
```suggestion
#if NETCOREAPP
```

Reply:
I think this is more honest, we are not depending on a feature from net6 or greater, just net core. Agreed?

Reply:
fixed